### PR TITLE
STCOM-898: Layout > Add textStart, textEnd properties

### PR DESCRIPTION
**Pull request by mariiaschv**
## Purpose
Adds new `text-start`, `text-end` classNames to `<Layout>` component to support both LTR and RTL languages.

## Issues
[STCOM-898](https://issues.folio.org/browse/STCOM-898)

### CODE REVIEW

**Review by zburke — APPROVED** [review]
LGTM, but I am not an RTL expert. @JohnC-80, what do you think?

**Review by Svitlana-Korol — APPROVED** [review]

**Review by JohnC-80 — APPROVED** [review]
Approved - Thanks!!

File: lib/Layout/stories/classNamePropertiesMapping.js [hunk 1]
```diff
@@ -41,5 +41,7 @@ export default {
   'right': 'float: right;',
   'textLeft': 'text-align: left;',
   'textCentered': 'text-align: center;',
-  'textRight': 'text-align: right;'
+  'textRight': 'text-align: right;',
+  'text-start': 'text-align: start;',
```
Comment (BogdanDenis) [inline lib/Layout/stories/classNamePropertiesMapping.js:45]: I think it's good to be consistent with names and use `textStart` and `textEnd` here

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: lib/Layout/stories/classNamePropertiesMapping.js (hunk 1 above)
Comment (mariiaschv) [reply lib/Layout/stories/classNamePropertiesMapping.js:45]: I agree with you about consistency but the reason I used `text-start` and `text-end` is [this comment](https://issues.folio.org/browse/STCOM-897?focusedCommentId=116278&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-116278) by @zburke where he said that 

> Kinda ugly that we have multiple naming conventions in place there, but I think we are leaning toward dash-separated-names now

But if you still think the names should be changed to camelCase I will do it, no problem :)

Comment (BogdanDenis) [reply lib/Layout/stories/classNamePropertiesMapping.js:45]: @mariiaschv oh I didn't notice that comment... then it's ok to merge as is

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**